### PR TITLE
README formatting + Docker infos

### PR DESCRIPTION
**Pull request by krptg0**
Switch readme to markdown, added docker info for user reference

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Doge
+Doge, a NSP collection manager
```
Comment (wowsuchdoge) [inline README.md]: I like it just saying Doge
```suggestion
Doge
```

File: README.md [hunk 2]
```diff
@@ -8,37 +8,45 @@ Doge
...
 - Install NodeJS LTS / 10 from https://nodejs.org/en/ (or `brew install node@10` on Mac) and restart your computer
-- Set your games folder root in /nut/conf/nut.conf (paths.scan, use / instead of \ even on Windows, ex: C:/games)
+  - Node 10, not 8, not 11
+- Set your games folder root in `/nut/conf/nut.conf` (paths.scan, use / instead of \ even on Windows, ex: C:/games)
+- If not on Windows, change the `PYTHON_CMD` to `python3` or whatever your python alias is
 - Run start.bat (Windows) or start.sh (Mac/Linux)
 - Visit the url you see in your web browser
 
+# Docker
+- Docker image is available at https://github.com/krptg0/doge
```
Comment (wowsuchdoge) [inline README.md]:
```suggestion
- Unofficial Docker image is available at https://github.com/krptg0/doge
```

File: README.md [hunk 3]
```diff
@@ -8,37 +8,45 @@ Doge
...
+- Contact krptg over Github for assistance, or open an issue directly on his repo
+
 # Credits
 - Doge
 - Thanks to blawar, morrison22, DBZTrunks, and more!
```
Comment (wowsuchdoge) [inline README.md]:
```suggestion
- Thanks to blawar, morrison22, krptg, Bob, DBZTrunks, and more!
```

File: README.md [hunk 4]
```diff
@@ -8,37 +8,45 @@ Doge
...
+- krptg for the Docker stuff
```
Comment (wowsuchdoge) [inline README.md]:
```suggestion
```

Included above

**Review by wowsuchdoge — CHANGES_REQUESTED** [review]
Having it in md will be nice, thanks!

**Comment by krptg0** [comment]
Looks good to me, I'll let you merge it ! (There's 1 change I can't... change, idk why)

**Review by krptg0 — COMMENTED** [review]
:+1:

**Pull request merged**